ui/dashboards/src/components/Variables/VariableEditorForm/variable-editor-form-model.ts

```diff
@@ -49,18 +49,14 @@ export type VariableEditorState = ReturnType<typeof getInitialState>;
 export function getVariableDefinitionFromState(state: VariableEditorState): VariableDefinition {
   const { name, title, kind } = state;
 
-  const commonSpec = {
-    name,
-    display: {
-      name: title,
-    } as Display,
-  };
+  const display: Display | undefined = title ? { name: title } : undefined;
```

Comment:
creating the field `display` like avoid creating an empty struct when the field `Display Label` is empty so avoid to trigger the backend verification on the `display` struct. (Validation that forbid the `name` to be empty)

Reply:
It looks like this is causing e2e's to fail, is everything working for you locally? I'm not sure why it can't find `data-testid="variable-editor"` anymore 😕 

```
✘  60 [chromium] › variables.spec.ts:45:7 › Dashboard: Variables › can add simple list variable (retry #2) (1m)
@perses-dev/e2e:e2e:ci: 
@perses-dev/e2e:e2e:ci: 
@perses-dev/e2e:e2e:ci:   1) [chromium] › variables.spec.ts:45:7 › Dashboard: Variables › can add simple list variable =====
@perses-dev/e2e:e2e:ci: 
@perses-dev/e2e:e2e:ci:     Test timeout of 60000ms exceeded.
@perses-dev/e2e:e2e:ci: 
@perses-dev/e2e:e2e:ci:     locator.evaluate: Target closed
@perses-dev/e2e:e2e:ci:     =========================== logs ===========================
@perses-dev/e2e:e2e:ci:     waiting for getByTestId('variable-editor')
```

Reply:
yeah that's quite weird. Well it's working fine when I'm creating manually a new variable. I'm looking at the e2e tests but not really sure if it's coming from me.

Reply:
looks like it was coming from the github action issues